docs/Community/Handling-a-New-Issue.md

```diff
@@ -0,0 +1,79 @@
+---
```

Comment:
rename the folder and file names to lower case `docs/community/handling-a-new-issue.md`